### PR TITLE
Create Cloud test project

### PR DESCRIPTION
**Pull request by BelehovEgor**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/327

### CODE REVIEW

File: CloudAPITestProject/Services/AuthService/ApiInformation.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿using CloudAPITestProject.Services.Abstract;
+using RestSharp;
+
+namespace CloudAPITestProject.Services.AuthService
+{
+    public class ApiInformation : ApiInformationCheck
+    {
+
```
Comment (KostinP) [inline CloudAPITestProject/Services/AuthService/ApiInformation.cs]: Extra empty line

**Review by KostinP — APPROVED** [review]

File: CloudAPITestProject/Services/AuthService/ApiInformation.cs (hunk 1 above)
Comment (BelehovEgor) [reply CloudAPITestProject/Services/AuthService/ApiInformation.cs]: fixed

**Review by MariyaSin — APPROVED** [review]

**Review by TIove — APPROVED** [review]

File: CloudAPITestProject/Services/ProjectService/ApiInformation.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+{
+    public class ApiInformation : ApiInformationCheck
+    {
+        protected override IRestResponse SendRequest()
+        {
+            var client = new RestClient("https://project.ltdo.xyz/apiinformation");
+            client.Timeout = -1;
+            var request = new RestRequest(Method.GET);
+            IRestResponse response = client.Execute(request);
+            return response;
```
Comment (djino97) [inline CloudAPITestProject/Services/ProjectService/ApiInformation.cs:14]: Add empty line

File: CloudAPITestProject/Services/CompanyService/ApiInformation.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+{
+    public class ApiInformation : ApiInformationCheck
+    {
+        protected override IRestResponse SendRequest()
+        {
+            var client = new RestClient("https://company.ltdo.xyz/apiinformation");
+            client.Timeout = -1;
+            var request = new RestRequest(Method.GET);
+            IRestResponse response = client.Execute(request);
+            return response;
```
Comment (djino97) [inline CloudAPITestProject/Services/CompanyService/ApiInformation.cs:17]: Add empty line

**Review by djino97 — APPROVED** [review]

File: CloudAPITestProject/Services/ProjectService/ApiInformation.cs (hunk 2 above)
Comment (BelehovEgor) [reply CloudAPITestProject/Services/ProjectService/ApiInformation.cs:14]: this is auto-generated code, I don't think it's worth "improving" it

File: CloudAPITestProject/Services/CompanyService/ApiInformation.cs (hunk 3 above)
Comment (BelehovEgor) [reply CloudAPITestProject/Services/CompanyService/ApiInformation.cs:17]: this is auto-generated code, I don't think it's worth "improving" it

**Pull request merged**